cmds/webboot/webboot.go

```diff
@@ -177,6 +188,10 @@ func main() {
 		verbose = log.Printf
 	}
 	cacheDir := *dir
+	if _, err := os.Stat(cacheDir); os.IsNotExist(err) {
+		log.Println("file does not exist")
```

Comment:
Where will this print come out? Will it mess up the UI? Does the UI have a way to present messages like this?

Reply:
I think the message will be print it the original terminal window that we run the webboot program, and after the program is exit we can see this message.